### PR TITLE
Add /health endpoint

### LINKED ISSUE #160: Add /health route

### PR DESCRIPTION
**Pull request by hectorgomezv**
Closes #160 

This route is implemented to match the exact behaviour of the current Rust client gateway.

### CODE REVIEW

File: src/routes/health/health.controller.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@ApiTags('health')
+@Controller({ path: 'health' })
+export class HealthController {
+  @ApiOkResponse({ type: String })
+  @Get()
+  getHealth(@Res() res: Response): void {
+    res.setHeader('Content-Type', 'application/json');
+    res.end('""');
+  }
+}
```
Comment (hectorgomezv) [inline src/routes/health/health.controller.ts:14]: @fmrsabino @moisses89 As I said in the description this implementation mimics the behaviour of the Rust client gateway, which returns `""` with a header `Content-Type: application/json`. This feels a bit odd, I don't know this is totally required. If we could return an object instead, the implementation/response would feel more natural in my opinion:

```javascript
  getHealth() {
    return { status: 'OK' };
  }
```

Comment (Uxio0) [reply src/routes/health/health.controller.ts:14]: I like your suggestion @hectorgomezv , and that shouldn't break anything as in liveness scripts we are only checking the `200` status code

Comment (hectorgomezv) [reply src/routes/health/health.controller.ts:14]: Great, that's what I thought, thanks @Uxio0!

Comment (fmrsabino) [reply src/routes/health/health.controller.ts:13]: Yeah for internal related routes we can go about these changes. I don't see any issue with the solution that you propose.

**Pull request merged**